sklearn/multiclass.py

```diff
@@ -269,6 +269,12 @@ class OneVsRestClassifier(
 
         .. versionadded:: 0.24
 
+    feature_names_in_ : ndarray of shape (`n_features_in_`,)
```

Comment:
`n_features_in_` without ticks? See https://github.com/scikit-learn/scikit-learn/pull/20818#discussion_r694676859.

Reply:
Without the ticks sphinx will give a warning: " WARNING: Unknown target name: "n_features_in"." because rst is considering it a [Hyperlink Target](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets)